td_biblio/views.py

```diff
@@ -4,7 +4,7 @@
 
 from django.contrib import messages
 from django.contrib.auth.decorators import login_required, user_passes_test
-from django.core.urlresolvers import reverse_lazy
+from django.urls import reverse_lazy
```

Comment:
Not sure this is compatible with older Django releases.

Reply:
Ah yes let me have a check for this